lms/templates/wiki/includes/cheatsheet.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-<section id="cheatsheetModal" class="modal" aria-hidden="true">
+<div id="cheatsheetModal" class="modal" aria-hidden="true">
```

Comment:
Hi @cptvitamin, the div immediately below this does have role=dialog and aria-labelledby. I'll try out aria-model=true, and also make sure the other 3 modals have all these attributes.